Library/Homebrew/brew.sh

```diff
@@ -12,10 +12,28 @@ then
   then
     export LC_ALL="en_US.UTF-8"
   else
-    export LC_ALL="C.UTF-8"
+    utf8_locales=$(locale -a | grep -e \.utf8 -e \.UTF-8)
+    if [[ -n $utf8_locales && 
+         ($utf8_locales =~ (^|[^a-zA-Z0-9_])(C|en_US)\.(utf8|UTF-8) ||
+          $utf8_locales =~ ^[a-zA-Z0-9_.]*) ]]
```

Comment:
My understanding is the format is always `[a-z][a-z]_[A-Z][A-Z].(utf|UTF-)8` so this could be a bit stricter.